hbp_knowledge/tau/miyata2011.bel

```diff
@@ -263,216 +330,296 @@ p(FPLX:HSP90) -> bp(GO:autophagy)
 member, and p23 are all critical for the transfer of kinases to Hsp90 and maturation of the active protein [76,105]"
+SET Confidence = "High"
 p(HGNC:CDC37) -> complex(a(MESH:"Protein Kinases"),p(FPLX:HSP90))
 p(HGNC:PTGES3) -> complex(a(MESH:"Protein Kinases"),p(FPLX:HSP90))
+UNSET Confidence
 
 SET Evidence = "In addition, Hsp90’s hydrolysis of ATP, which is stimulated by Aha1, facilitates
 polypeptide release [82,106] and transfer to CHIP or other E3 ligases [107,108]"
+SET Confidence = "High"
 p(HGNC:AHSA1) -> rxn(reactants(a(CHEBI:ATP)),products(a(CHEBI:ADP),a(CHEBI:"phosphate(3-)")))
 p(HGNC:AHSA1) -> complex(a(CHEBI:polypeptide),p(HGNC:STUB1))
+UNSET Confidence
 
 SET Evidence = "Although both Hsp70 and Hsp90 can promote degradation of client proteins, it has
 recently been shown that, functionally, the Hsp70 complex often dominates triage decisions [85,107,109]"
+SET Confidence = "High"
 p(INTERPRO:"Heat shock protein 70 family") -> bp(GO:"protein catabolic process")
 p(FPLX:HSP90) -> bp(GO:"protein catabolic process")
+UNSET Confidence
 
 SET Evidence = "However, under potentially proteotoxic conditions, the
 post-translational modifications or mutations that damage tau’s affinity for microtubules and
 favor its aggregation are thought to generate a molecular ‘danger signal’ that specifically alerts
 the quality control system [112,113]."
+SET Confidence = "Medium"
 p(HGNC:MAPT,var("?")) -> a(CONSO:"Tau aggregates")
 p(HGNC:MAPT,var("?")) -| complex(p(HGNC:MAPT),a(GO:microtubule))
+p(HGNC:MAPT,var("?")) -- bp(GO:"protein quality control for misfolded or incompletely synthesized proteins")
+UNSET Confidence
 
 SET Evidence = "Hsp70 has been shown to both stabilize binding
 of tau to microtubules [114] and promote its degradation in combination with CHIP [115,116]"
+SET Confidence = "High"
 p(INTERPRO:"Heat shock protein 70 family") -> complex(p(HGNC:MAPT),a(GO:microtubule))
 complex(p(HGNC:STUB1),p(INTERPRO:"Heat shock protein 70 family")) -> deg(p(HGNC:MAPT))
+UNSET Confidence
 
 SET Evidence = "In addition, recent work has demonstrated that Hsc70, the constitutive cytosolic form of
 Hsp70s, also dynamically regulates the association of tau with microtubules"
+SET Confidence = "High"
 p(HGNC:HSPA8) reg complex(p(HGNC:MAPT),a(GO:microtubule))
+UNSET Confidence
 
 SET Evidence = "In fact, recent work from our group has shown that inhibition of the ATPase
 activity of Hsp70/Hsc70 promotes proteasomal degradation of tau; whereas activation results
 in tau accumulation [117]"
-act(p(INTERPRO:"Heat shock protein 70 family")) -> deg(p(HGNC:MAPT))
-act(p(HGNC:HSPA8)) -> deg(p(HGNC:MAPT))
+SET Confidence = "Low"
+act(p(INTERPRO:"Heat shock protein 70 family"),ma(GO:"ATPase activity")) -| deg(p(HGNC:MAPT))
+act(p(HGNC:HSPA8),ma(GO:"ATPase activity")) -| deg(p(HGNC:MAPT))
+act(p(INTERPRO:"Heat shock protein 70 family")) -> p(HGNC:MAPT)
+act(p(HGNC:HSPA8)) -> p(HGNC:MAPT)
+UNSET Confidence
```

Comment:
@sgebel @cthoyt please check for these statements.

Reply:
please add ATPase activity to the HSPs in all statements, i.e. line 393 and line 394;